lib/internal/Magento/Framework/App/Http.php

```diff
@@ -150,7 +150,7 @@ public function catchException(Bootstrap $bootstrap, \Exception $exception)
      */
     private function handleDeveloperMode(Bootstrap $bootstrap, \Exception $exception)
     {
-        if ($bootstrap->isDeveloperMode()) {
+        if (!$bootstrap->isDeveloperMode()) {
```

Comment:
why was this changed? This block should only be executed if we are in developer mode.